datahub-web-react/src/app/domain/DomainsList.tsx

```diff
@@ -64,12 +64,12 @@ export const DomainsList = () => {
                 query,
             },
         },
-        fetchPolicy: 'no-cache',
+        fetchPolicy: 'cache-first',
```

Comment:
interesting, nice. I know there's been bugs with using apollo cache before, but idk if that's how we were using it or if it's just difficult to work with. If it's working well for domains then we can at the very least extend this to other similar situations! (ie. the glossary)

Reply:
Seems to be working!